### ISSUE TITLE
epi-genregress fails on some subjects

### ISSUE THREAD

**Issue by josephdviviano** [josephdviviano/epitome#55]
`ERROR: failed to find the top principal components of input data:`

**Comment by josephdviviano** [comment]
generating a test epitome subject that fails (`DTI_CMH_S036_01`) so I can debug.

**Comment by josephdviviano** [comment]
Looks like forcing randomized PCA on these subjects fixes the issue. Not ideal, but for our use case the difference in the algorithms seems non-significant. See b7c02442a356993e52da38b4df00893a0f18b9eb

I'm going to continue monitoring for problems, but consider this closed for now.

I found [this bug](http://stackoverflow.com/questions/21827594/raise-linalgerrorsvd-did-not-converge-linalgerror-svd-did-not-converge-in-m) that suggests my normal PCA failing (was specifically 'SVD did not converge') was due to the particular version of scipy I'm using [see this answer](http://stackoverflow.com/a/25738781). Confirmed my data had non `nans` or `infs`.
